sorc/ncep_post.fd/TIMEF.f

```diff
@@ -3,17 +3,20 @@
 !mp     HPs)  Designed to duplicate timef (elapsed time in milliseconds)
 !
         function timef()
```

Comment:
@GeorgeVandenberghe-NOAA There are timer updates in UPP code recently. Would these timer updates be used for your testing only or other intentions?

Reply:
Additional timer codes are only for my testing.  But TIMEF.f  itself is not producing valid times on some systems and I fixed that in passing.   MPI_WTIME provides the highest resolution timer on our intel based systems; much better than rtc()